library/src/hooks/places-service.ts

```diff
@@ -21,8 +29,29 @@ export const usePlacesService = (): google.maps.places.PlacesService | null => {
       );
     }
 
-    return new google.maps.places.PlacesService(map);
-  }, [map]);
+    if (props?.divElement === undefined) {
```

Comment:
> Hm, good point! Needed a minute to understand the scenario.. but it is actually a thing. 👍

Maybe you can add a comment, so it's clear to everyone.

Reply:
Like this? ( I am not very creative with the comment right now - I know)

Reply:
How about something like ...
```js
    // Create places service which renders attributions in the map container
    if (props?.divElement === undefined) {
      // Wait for map to be initialized
      if (!map) {
        return;
      }

      // ...
    }

    // Create places service which renders attributions in the passed div element
    // Wait for div element to be available
    if (!props?.divElement) {
      // ...
    }
```

... or ...

```js
    if (props?.divElement === undefined) {
      // Create places service which renders attributions in the map container
      // Wait for map to be initialized
      if (!map) {
        return;
      }

      // ...
    } else {
      // Create places service which renders attributions in the passed div element
      // Wait for div element to be available
      if (!props?.divElement) {
        return;
      }

      // ...
    }
```

Reply:
Ah okay, I thought you wanted one specifically for the `div===undefined`. Yep, this one makes sense 👍